Ports/lrzip/patches/0001-Remove-Linux-specific-code.patch

```diff
@@ -0,0 +1,36 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Fabian Dellwing <fabian.dellwing@gmail.com>
+Date: Sat, 11 Mar 2023 14:44:17 +0100
+Subject: [PATCH] Remove Linux 0 specific code
+
+---
+ main.c | 9 ++-------
+ 1 file changed, 2 insertions(+), 7 deletions(-)
+
+diff --git a/main.c b/main.c
+index eb0985feb1f537533f9f6a983fa3c0e39de730bf..107f25d5ff477977c6ed042c1ff0a50235fbe121 100644
+--- a/main.c
++++ b/main.c
+@@ -119,7 +119,7 @@ static void usage(bool compat)
+ 	}
+ 	if (!compat)
+ 		print_output("	-L, --level level	set lzma/bzip2/gzip compression level (1-9, default 7)\n");
+-	print_output("	-N, --nice-level value	Set nice value to value (default %d)\n", compat ? 0 : 19);
++	print_output("	-N, --nice-level value	Set nice value to value (default %d) [This does nothing on SerenityOS]\n", compat ? 0 : 19);
+ 	print_output("	-p, --threads value	Set processor count to override number of threads\n");
+ 	print_output("	-m, --maxram size	Set maximum available ram in hundreds of MB\n");
+ 	print_output("				overrides detected amount of available ram\n");
+@@ -449,12 +449,7 @@ int main(int argc, char *argv[])
+ 				failure("Extra characters after ramsize: \'%s\'\n", endptr);
+ 			break;
+ 		case 'N':
+-			nice_set = true;
+-			control->nice_val = strtol(optarg, &endptr, 10);
+-			if (control->nice_val < PRIO_MIN || control->nice_val > PRIO_MAX)
+-				failure("Invalid nice value (must be %d...%d)\n", PRIO_MIN, PRIO_MAX);
+-			if (*endptr)
+-				failure("Extra characters after nice level: \'%s\'\n", endptr);
++			/* NOP for SerenityOS */
```

Comment:
Same here, helps with upstreaming in case we ever want to do that. :^)

```suggestion
+#ifndef __serenity__
 			nice_set = true;
 			control->nice_val = strtol(optarg, &endptr, 10);
 			if (control->nice_val < PRIO_MIN || control->nice_val > PRIO_MAX)
 				failure("Invalid nice value (must be %d...%d)\n", PRIO_MIN, PRIO_MAX);
 			if (*endptr)
 				failure("Extra characters after nice level: \'%s\'\n", endptr);
+#endif
```